experiments/opt_impact.sh

```
@@ -0,0 +1,462 @@
+echo "flag,input,baseline_jam,delta_jam,baseline_code,delta_code,compile_ok" > "$DELTAS_CSV"
+
+# -----------------------------------------------------------------------------
+# Phase A: baseline compile for every input
+# -----------------------------------------------------------------------------
+echo "==> Phase A: baseline (all opts on)"
+declare -a BL_JAM BL_CODE BL_OK
+for i in "${!KEPT_INPUTS[@]}"; do
+  entry="${KEPT_INPUTS[$i]}"
+  IFS='|' read -r name source imports adapter <<< "$entry"
+  out="$TMP_JAM_DIR/baseline-$name.jam"
+  extra=""
```

Shellcheck (0.11.0)

[warning] Variable was used as an array but is now assigned a string.

(SC2178)